### PR TITLE
Adding support to luigi version

### PR DESCRIPTION
**Pull request by gmendonca**
Yo! If the latest version has some breaking changes, sometimes is useful to have the option to pin a version. Here we go.

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -68,7 +68,15 @@
 python_pip 'boto'
 python_pip 'python-daemon'
 python_pip 'SQLAlchemy'
-python_pip 'luigi'
+
+if node['luigi']['version']
+  python_pip 'luigi' do
+    version node['luigi']['version']
+    :upgrade
```
Comment (barata) [inline recipes/default.rb]: Does it work if you don't have a previous version installed?

Comment (barata) [reply recipes/default.rb]: Anyway, I would remove `if` and always require a version to be specified.

**Comment by gmendonca** [comment]
@barata @adriana-penaranda

File: README.md [hunk 2]
```diff
@@ -93,7 +99,7 @@ In order to load ENV variables when running your service, you can add them as:
 default['luigi']['server']['env']['VARIABLE'] = 'value'
 ```
 
-In order to create a default luidid directory, you can add:
+In order to create a default luidid directory you can add:
```
Comment (barata) [inline README.md]: _luidid_ typo.

Comment (gmendonca) [reply README.md]: Good catch!

**Comment by gmendonca** [comment]
@barata @adriana-penaranda

**Review by barata — APPROVED** [review]

**Pull request merged**